jlrs/tests/eval_string.rs

```diff
@@ -0,0 +1,56 @@
+use jlrs::prelude::*;
```

Comment:
Your tests look great. If you could checkout the v0.8-branch and add these tests, I'd happily accept that pull request.